packages/devtools_testing/lib/support/memory_test_data.dart

```diff
@@ -0,0 +1,8763 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: implementation_imports
+import 'dart:convert';
+
+import 'package:devtools_app/src/utils.dart';
+
+import 'test_utils.dart';
+
+/// Little over a 1 minute run starting at 6:08:09 till 6:09:15
+/// Contains 2 manual snapshots, 3 auto snapshots, 2 manual GCs,
+/// Also contains, 3 allocation profile 2 groups of:
+///     - allocation profile followed by an allocation profile reset
+///     - allocation profile only
+/// 70 VM GC events
+/// Dart Heap capacity ~684 MB, Used ~42 MB, External 634 MB, and RSS 534 MB
+/// 292 samples
+/// 
+/// How to create a new test data set. Run DevTools collecting stats then press
+/// the Export button. Then copy the contents of the exported file and assign to
+/// testHeapSampleData.
+final testHeapSampleData = {
```

Comment:
you can use a `r'''` string to make this easier for the analysis server to handle and make the test data match the actual parsed json a little more.

Reply:
Good idea done.